integration_tests/mockApis/curiousApi.ts

```diff
@@ -73,18 +73,9 @@ export default {
       },
     })
   },
-  stubGetCuriousGoals: (options?: { prisonerNumber?: string; emptyStates?: boolean }) => {
-    const prisonerNumber = options?.prisonerNumber || 'G6123VU'
-    const responseBody =
-      !options || options.emptyStates === null || options.emptyStates === undefined || options.emptyStates === false
-        ? aValidLearnerGoals({ prn: prisonerNumber })
-        : aValidLearnerGoals({
-            prn: prisonerNumber,
-            employmentGoals: [],
-            personalGoals: [],
-            longTermGoals: [],
-            shortTermGoals: [],
-          })
+
+  stubGetCuriousGoals: (prisonerNumber = 'G6123VU') => {
```

Comment:
I've changed this `stubGetCuriousGoals` stub so that it returns data that represents a prisoner's goals
I have added separate stubs below for a prisoner with no goals, and a 500 error
That way the cypress test itself has to be explicit about which stub it wants to use, which increases the readability of the test 👍